src/pages/pupils/programmes/[programmeSlug]/options/examboard/[examboardSlug].tsx

```diff
@@ -1,73 +1,26 @@
 import { GetStaticProps, GetStaticPropsResult } from "next";
 
-import { PupilViewsProgrammeListing } from "@/components/PupilViews/PupilProgrammeListing/PupilProgrammeListing.view";
 import {
-  ProgrammesPageProps,
   OptionsURLParams,
-  isExamboardSlug,
-  getYearSlug,
+  getPupilOptionData,
 } from "@/pages-helpers/pupil/options-pages/options-pages-helpers";
-import OakError from "@/errors/OakError";
 import { getStaticPaths as getStaticPathsTemplate } from "@/pages-helpers/get-static-paths";
-import curriculumApi2023 from "@/node-lib/curriculum-api-2023";
+import { PupilViewsProgrammeListingProps } from "@/components/PupilViews/PupilProgrammeListing/PupilProgrammeListing.view";
 
 // This page is for moving back through pupil journey from subject with examboards and tiers
 
-const ProgrammesWithExamboardPage = ({
-  programmes,
-  baseSlug,
-  yearSlug,
-  examboardSlug,
-}: ProgrammesPageProps) => {
-  return (
-    <PupilViewsProgrammeListing
-      programmes={programmes}
-      baseSlug={baseSlug}
-      yearSlug={yearSlug}
-      examboardSlug={examboardSlug}
-    />
-  );
-};
+export { PupilViewsProgrammeListing as default } from "@/components/PupilViews/PupilProgrammeListing/PupilProgrammeListing.view";
```

Comment:
`/options.tsx` & `/options/examboard/[examboardSlug].tsx` can now just export the PupilViewsProgrammeListing as default. Far less duplication of code.